otx/algorithms/segmentation/adapters/mmseg/datasets/dataset.py

```diff
@@ -257,12 +261,16 @@ def __init__(self, **kwargs):
         if test_mode is False:
             self.img_indices = get_old_new_img_indices(classes, new_classes, otx_dataset)
 
+        for pipe in pipeline:
+            if pipe["type"] == "LoadImageFromFile" and "use_otx_adapter" in pipe:
+                use_otx_adapter = pipe["use_otx_adapter"]
+
```

Comment:
Please add `break` here inside the if-statement. It doesn't need to iterate all for updating use_otx_adapter.